apps/condo/domains/organization/sbbol/common.js

```diff
@@ -0,0 +1,61 @@
+const Ajv = require('ajv')
+const SBBOL_IMPORT_NAME = 'sbbol'
+const SBBOL_SESSION_KEY = 'sbbol'
+
+const EXAMPLE_USER_INFO = {
```

Comment:
Please, move it to testSchema, otherwise, we will have mess of working code and testing data ;)

Reply:
Done